### ISSUE TITLE
More powerful importing/exporting from another project systems

### ISSUE THREAD

**Issue by VISTALL** [consulo/consulo#249]
* First problem with VS solutions. In one folder you can create unlimited solutions, but you can have only one Consulo project
   - Provide unstable module setup? without saving **modules.xml** file
   - allow select in recent projects - solution file (not consulo project)
 * ...

**Comment by brogan89** [comment]
I was just about to create a new issue for this. I was unable to export my settings at all. The .jar file was never saved. But I was hoping for a more seamless version like with Ride how you can create a repo for your settings. Its a very handy was of syncing settings across multiple devices.

I'd love to help out with this project because I am really liking the potential of this IDE. But I have very little java knowledge.

**Comment by VISTALL** [comment]
Hello. I found this problem (can't export any IDE settings) yesterday. It's my failure. I will fix it in near future (next week)

Thanks

**Comment by VISTALL** [comment]
@brogan89 Hello again. I fixed this issue. Need some time until all my projects rebuild it (https://ci.consulo.io/)

Later you need change update channel to **Nightly** and call **Check for update** (https://github.com/consulo/consulo/wiki/Update-channels)

**Comment by brogan89** [comment]
that was fast! :smile: thank you. Unfortunately changing to nightly build has caused some issues for me. I can make a new issue with this. but it has broken my Unity project too. saying TMPro could not be found. Usually in Unity packages. But the package manager is missing the Windows tab in Unity editor. Did i do something wrong?

**Comment by VISTALL** [comment]
Yeah. You need go to consulo-unity3d, and report issue. If you have TMPro issues - it's unity packages problem. It's pain in my ass, due i need reimplement Unity package system resolving (they changing package paths around 3-4 times).

I think you need Re-import unity project (right click in project view at root directory) - it may helps.
